morpho5/vm/vm.c

```diff
@@ -1497,7 +1503,7 @@ bool morpho_interpret(vm *v, value *rstart, instructionindx istart) {
             if (v->ehp<v->errorhandlers) v->ehp=NULL;
             DISPATCH();
         
-        CASE_CODE(ARRAY):
+        /*CASE_CODE(ARRAY):
```

Comment:
Can we just delete this?

Reply:
I plan to delete it after we carefully re-examine the VM's instruction set [there are a few design decisions to re-examine]. 

The new Array implementation is a bit slower (~20%), because: 
* It calls Array to instantiate the array, which is obviously slower than an opcode (but significantly more general). 
* I removed some loop unrolling to make array_getelement and array_setelement more readable. 

My expectation is that we'll recover this performance and more from planned work in the optimizer. I thought a lot of the optimizations in place were probably premature and needed a more systematic approach.

Reply:
Sounds great, thanks for the info.